Social Distancing Measures.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>What is the thought process behind using <code>mean</code>  of all the simulations? Am I interpreting the code+plot correctly?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/coronaIndia/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>The time steps for SEIRSPLUS is less than 1. If we look at model.tseries, we will be able to see the exact time period for the model is predicting infections. Let's say, on day 14, there are multiple time periods for which we have S, E, D_E, D_I, I,  R, F with sufficient std dev. Therefore, I take mean to get a stable value for that day.</p>